cypress/e2e/searchForm.cy.ts

```diff
@@ -0,0 +1,19 @@
+describe("SearchForm", () => {
+  it("submits the form with proper search query", () => {
+    const searchQuery = "Test search query";
+    cy.visit("http://localhost:3000/", {
+      onBeforeLoad(win) {
+        cy.stub(win.console, "log").as("consoleLog");
+        cy.stub(win.console, "error").as("consoleError");
+      },
+    });
+
```

Comment:
cy.get('input[type="text"]').should('exist');
    cy.get('button[type="submit"]').should('exist');
    
    Nedd to check if there are the elements for interaction

Reply:
Done